src/places_api.js

```diff
@@ -14,7 +14,7 @@ function findCities(botContext) {
   return client
     .findPlaceFromText({
       params: {
-        key: process.env.GOOGLE_MAPS_API_KEY2,
+        key: process.env.GOOGLE_MAPS_API_KEY3,
```

Comment:
I suppose you don't need to pass keys. I propose the following: Have a pull of keys available, and just randomly take one of it for each of usage. It is veird that you checks `KEY3` then use `if (KEY2)` and then using KEY3. 

Create a function getGoogleApiKey() that will return a random one